app/services/friends_service.rb

```diff
@@ -0,0 +1,21 @@
+class FriendsService
+  class << self
+    def invite(emails)
+      @incorrect_emails = ''
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.